dotnet/src/SemanticKernel/Orchestration/SKFunctionExtensions.cs

```diff
@@ -35,7 +23,8 @@ public static ISKFunction UseCompletionSettings(this ISKFunction skFunction, Com
     /// <returns>Self instance</returns>
     public static ISKFunction UseMaxTokens(this ISKFunction skFunction, int maxTokens)
     {
```

Comment:
nit: "maxTokens cannot be null"

Might also want to have a check for "maxTokens must be greater than 0"